task/mongo-tasks.js

```diff
@@ -251,7 +601,60 @@ async function task_1_18(db) {
  *       - do not hesitate to "ensureIndex" in "before" function at the top if needed https://docs.mongodb.com/manual/reference/method/db.collection.ensureIndex/
  */
 async function task_1_19(db) {
-    throw new Error("Not implemented");
+    db.collection('orders').createIndex({"CustomerID": 1});
```

Comment:
The indexes here can't improve result in tests because we need time to create indexes. use `function before` for that